R/page.R

```diff
@@ -38,7 +39,9 @@ page_fluid <- function(..., title = NULL, theme = bs_theme(), lang = NULL) {
   )
 }
 
-#' @rdname page
+#' @describeIn page A \pkg{bslib} wrapper for [shiny::fixedPage()], a fixed
+#'   Bootstrap-based page layout where the page content container is centered
+#'   horizontally and its width is constrained.
 #' @inheritParams shiny::fixedPage
 #' @export
 page_fixed <- function(..., title = NULL, theme = bs_theme(), lang = NULL) {
```

Comment:
Thanks for this. Can we also add a `@describeIn`  for `page_navbar()`?

Reply:
Also, once we have `@describeIn` for all the pages, we could/should update the `@description` as well, especially since it's a bit out-of-date now. How about something quite simple like?

> Modern Bootstrap page layouts.

Reply:
👍